context/src/junit/validator.rs

```diff
@@ -115,8 +115,8 @@ pub fn validate(report: &Report) -> JunitReportValidation {
             if let Some(raw_test_case_id) = test_case.extra.get("id") {
                 let test_case_id = uuid::Uuid::parse_str(raw_test_case_id).unwrap_or_default();
                 if test_case_id.get_version() != Some(uuid::Version::Sha1) {
-                    test_case_validation.add_issue(JunitValidationIssue::Invalid(
-                        JunitTestCaseValidationIssueInvalid::TestCaseInvalidId(
```

Comment:
Demote it to suboptimal. `ID` can be set by unrelated junit runners and it's better to just have the ETL / service handle generating it in general.